doc/source/ray-core/tasks/scheduling.rst

```diff
@@ -87,35 +87,13 @@ especially in a multi-tenant cluster: for example, an application won't know wha
 
 Locality-Aware Scheduling
 -------------------------
-When the scheduling strategy is ``"DEFAULT"``, Ray also prefers nodes that have large task arguments locally
-to avoid transferring data over the network.
-If there are multiple large task arguments, the node with most object bytes local is preferred.
+By default, Ray prefers available nodes that have large task arguments locally
+to avoid transferring data over the network. If there are multiple large task arguments,
+the node with most object bytes local is preferred.
```

Comment:
Does this always take precedence over the default packing scheduling? Would be good to clarify that.